modules/ROOT/pages/rest-sdk/edits/scaffold-connector-descriptor.adoc

```diff
@@ -0,0 +1,84 @@
+= Scaffold a Connector Descriptor
+ifndef::env-site,env-github[]
+include::../_attributes.adoc[]
+endif::[]
+
+To create a connector using REST SDK, you must first scaffold a connector
+descriptor. Use a MOJO to scaffold a connector descriptor from an API specification.
+
+== Connector Descriptor
+
+A connector descriptor is a YAML file that enables you to describe a
+connector, customize aspects of the generated connector (such as name,
+Maven GAV, operation names, layout information, and so on), and provide
+additional metadata that is specific to MuleSoft’s connectors.
+
+When creating a connector descriptor, do not edit the API
+specification. The connector descriptor you create has the same functionality
+as your API specification. If you modify your API specification and a new
+version of an API specification is released, move all the changes from the old version
+to the new version. API specifications must be as complete as possible and provide
+rich schemas for every input or output type, example, security information, and
+so on.
+
+== Scaffold the Connector Descriptor
+
+To scaffold the connector descriptor, run the `createDescriptor` MOJO:
+
+----
+mvn com.mulesoft.connectivity:rest-sdk-mojo:<PLUGIN_VERSION>:createDescriptor -DapiSpec=<PATH_TO_SPEC>/api.raml
+----
+
+* Where `<PATH_TO_SPEC>` is the path to the location of your API specification.
+* Where `<PLUGIN_VERSION>` is the version of the REST SDK plugin you want to
+use. If you omit this parameter, the latest version is used.
```

Comment:
I'm guessing I don't understand something here that I should, so please let me know when you get a chance, so that I can learn.

Reply:
I'm glad you asked -- the parameters in the table can be added to the command by using the -D prefix. Only apiSpec is required while the other parameters are not, so that's why the apiSpec parameter is shown in the example command: 
mvn com.mulesoft.connectivity:rest-sdk-mojo:<PLUGIN_VERSION>:createDescriptor **-DapiSpec**=<PATH_TO_SPEC>/api.raml